client/blocks/comments/post-comment-content.jsx

```diff
@@ -8,7 +8,11 @@ export default class PostCommentContent extends React.Component {
 	render() {
 		// Don't trust comment content unless it was provided by the API
 		if ( this.props.isPlaceholder ) {
-			return <div className="comments__comment-content">{ this.props.content }</div>;
+			return <div className="comments__comment-content">{
+				this.props.content.split( '\n' ).map( ( item, key ) => {
+					return <span key={ key }>{ item }<br /></span>;
```

Comment:
`<p>` tags around each line gives double spacing. Regular comments usually have one `<p>` with a `<br />` after each line. I got the span/br thing from [this Medium post](https://medium.com/@kevinsimper/react-newline-to-break-nl2br-a1c240ba746).